src/Vim.xml

```diff
@@ -0,0 +1,120 @@
+<SPDX name="Vim License" identifier="Vim" osi-approved="false">
```

Comment:
Seems to be indent misread based on the text file spacing, all we have to do is nest the actual <list> tags. Will do that and update.